apps/emqx/src/emqx_tls_lib.erl

```diff
@@ -432,15 +406,15 @@ save_pem_file(Dir, KeyPath, Pem, DryRun) ->
 %% the filename is prefixed by the option name without the 'file' part
 %% and suffixed with the first 8 byets the PEM content's md5 checksum.
 %% e.g. key-1234567890abcdef, cert-1234567890abcdef, and cacert-1234567890abcdef
-is_generated_file(Filename) ->
+is_managed_ssl_file(Filename) ->
     case string:split(filename:basename(Filename), "-") of
         [_Name, Suffix] -> is_hex_str(Suffix);
         _ -> false
     end.
 
 pem_file_name(Dir, KeyPath, Pem) ->
     <<CK:8/binary, _/binary>> = crypto:hash(md5, Pem),
```

Comment:
Maybe add the original full path together with pem content to make the md5 hash, so to make more unique suffix. Because file base name

Reply:
Oh, right, forgot about that. Made suffixes random in [d5aedaa](https://github.com/emqx/emqx/pull/10927/commits/d5aedaac7daa9237f3af71f3fe6532feac7c829d).